pgjdbc/src/main/java/org/postgresql/jdbc/TypeInfoCache.java

```diff
@@ -666,17 +679,17 @@ public synchronized String getJavaClass(int oid) throws SQLException {
   }
 
   public String getTypeForAlias(String alias) {
-    String type = typeAliases.get(alias);
+    String type = TYPE_ALIASES.get(alias);
     if (type != null) {
       return type;
     }
-    if (alias.indexOf('"') == -1) {
-      type = typeAliases.get(alias.toLowerCase());
-      if (type != null) {
-        return type;
-      }
+    type = TYPE_ALIASES.get(alias.toLowerCase());
```

Comment:
Should this specify Locale.US?

Reply:
Maybe this question should be asked for the entirety of the class. There are many places that String.toLowerCase() is called without providing a Locale.